public/static/docs/api-reference/index.md

```diff
@@ -0,0 +1,30 @@
+# Python API
+
+When you [install](/doc/install) DVC with an package manager like `pip` or
```

Comment:
simplify - "if you need to use DVC API you can consider DVC as a library, install it like with pip or conda (see installation) and into your project dependencies if needed"

Reply:
Refactored to 
```markdown
DVC can be used as a Python library, simply [install](/doc/install) it with a
package manager like `pip` or `conda`, and as a Python
[project requirement](https://pip.pypa.io/en/latest/user_guide/#requirements-files)
if needed. This reference provides the details about the functions in the API
module `dvc.api`, which can be imported any regular way, for example:
```